README.de.md

```diff
@@ -0,0 +1,590 @@
+# <img src="images/pyxel_logo_152x64.png">
+
+[ [English](README.md) | [中文](README.cn.md) | [Español](README.es.md) | [Français](README.fr.md) | [Italiano](README.it.md) | [日本語](README.ja.md) | [한국어](README.ko.md) | [Português](README.pt.md) | [Русский](README.ru.md) | [Deutsch](README.de.md)]
+
+**Pyxel** ist eine Retro-Spielengine für Python.
+
+Dank seiner einfachen, von Retro-Spielkonsolen inspirierten, Spezifikationen, wie z. B. dass nur 16 Farben angezeigt werden können und nur 4 Töne gleichzeitig wiedergegeben werden können, kannst du dich frei fühlen, um Spiele im Pixel-Art-Stil zu entwickeln.
+
+<a href="pyxel/examples/01_hello_pyxel.py" target="_blank">
+<img src="pyxel/examples/screenshots/01_hello_pyxel.gif" width="48%">
+</a>
+
+<a href="pyxel/examples/02_jump_game.py" target="_blank">
+<img src="pyxel/examples/screenshots/02_jump_game.gif" width="48%">
+</a>
+
+<a href="pyxel/examples/03_draw_api.py" target="_blank">
+<img src="pyxel/examples/screenshots/03_draw_api.gif" width="48%">
+</a>
+
+<a href="pyxel/examples/04_sound_api.py" target="_blank">
+<img src="pyxel/examples/screenshots/04_sound_api.gif" width="48%">
+</a>
+
+<a href="pyxel/editor/screenshots/image_tilemap_editor.gif" target="_blank">
+<img src="pyxel/editor/screenshots/image_tilemap_editor.gif" width="48%">
+</a>
+
+<a href="pyxel/editor/screenshots/sound_music_editor.gif" target="_blank">
+<img src="pyxel/editor/screenshots/sound_music_editor.gif" width="48%">
+</a>
+
+Die Spezifikationen der Spielkonsole und der APIs für Pyxel beziehen sich auf die großartigen [PICO-8] (https://www.lexaloffle.com/pico-8.php) und [TIC-80] (https://tic.computer/).
+
+Pyxel ist quelloffen und kostenlos zu verwenden. Fang an, ein Retro-Spiel mit Pyxel zu entwickeln!
+
+## Spezifikationen
+
+- Läuft unter Windows, Mac, und Linux
+- Code schreiben mit Python3
+- Festgelegte 16-Farben-Palette
+- 256x256 große 3 Image Banks
+- 256x256 große 8 Tilemaps
+- 4 Kanäle mit 64 definierbaren Tönen
+- 8 Musikspuren, die beliebige Klänge kombinieren können
+- Tastatur-, Maus- und Gamepad-Eingaben
+- Bild- und Toneditor
+
+### Farbpalette
+
+<img src="pyxel/examples/screenshots/05_color_palette.png">
+<br><br>
+<img src="images/pyxel_palette.png">
+
+## Installationsanleitung
+
+### Windows
+
+Zuerst installiere [Python3](https://www.python.org/) (Version 3.6.8 oder höher).
+
+Wenn du Python mit dem offiziellen Installationsprogramm installiest, **füge Python zu PATH** hinzu, indem du das Feld unten auswählst:

```

Comment:
Seems funny to exclude `hinzu` from the highlight

Reply:
Sry, gonna fix when i come home